integrations/extensions/starter-kits/servicenow/assets/sn.postman.json

```diff
@@ -0,0 +1,186 @@
+{
+	"info": {
+		"_postman_id": "5a4e921c-d0a2-49b6-867e-9c94cf6d3054",
+		"name": "ServiceNow Blog",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "https://dev93610.service-now.com/api/now/table/incident",
+			"request": {
+				"auth": {
+					"type": "basic",
+					"basic": [
+						{
+							"key": "password",
+							"value": "Rest!999",
```

Comment:
This isn't your actual password, right :)

Reply:
The instance will be cleared in 10 days anyhow... :)